### ISSUE TITLE
Audio alerts error

### ISSUE THREAD

**Issue by Acimus** [greenc/CookieMaster#27]
Audio alerts aren't working in the latest version, I'm using chrome extension. Both default and custom links report an error.

![image](https://f.cloud.github.com/assets/6691666/2178518/7aeb67f0-966b-11e3-9e29-65655de93fb9.png)

**Comment by greenc** [comment]
Does this happen repeatedly, even when you refresh the game/plugin?

I can't reproduce this myself :/

**Comment by Acimus** [comment]
Yes, even after browser restart.  I had an update notification an hour ago, but it didn't update somehow..

**Comment by greenc** [comment]
Weird. Does the direct link to the audio file play correctly?

http://cookiemaster.co.uk/assets/gc.mp3

**Comment by Acimus** [comment]
Plays just fine in the browser's tab, still nothing in the cookieclicker tab.  :/

**Comment by Nobody1707** [comment]
I'm having the same issue with the userscript in Opera and Violentmonkey.

**Comment by worldendDominator** [comment]
I'm having the same problem, but in this case I'm using a local file.
Just in case: I use noScript, but file:/// is allowed.

**Comment by greenc** [comment]
Can you try running the following, replacing `url` with your local file:

``` javascript
(function() {
    var url = 'http://cookiemaster.co.uk/assets/gc.mp3';
    var test = new Audio(url);
    test.play();
})()
```

**Comment by worldendDominator** [comment]
How do I run that? Putting it into page address opens Google search, and I can't create a bookmark with it (OK button dimmed, not a valid address?).

**Comment by greenc** [comment]
Run it from your browser's console (F12 or Ctrl+Shift+i to open). Let me know what happens.

**Comment by worldendDominator** [comment]
<-
(function() {
    var url = 'file:///D:/clash.mp3';
    var test = new Audio(url);
    test.play();
})()

->
undefined

And nothing else. Putting your script (with the default sound) plays the sound fine.

**Comment by adamsalem** [comment]
Can replicate this issue in Chromium, not in Chrome. My Chromium is SRWare Iron.

The function you suggested works in Chrome (i.e. the sound plays but returns undefined) but not in Iron... Weird!

I think this might be a HTML5 audio issue in Iron, I have a site that uses multiple HTML5 audio Elements, Iron does not play any of them but Chrome plays them all fine.
